src/Core/src/Handlers/HybridWebView/HybridWebView.ts

```diff
@@ -0,0 +1,255 @@
+/*
```

Comment:
Noob question, but why do we have both the HybridWebView.js and the HybridView.ts file? Aren't they essentially the same thing?